test/parallel/test-fs-watch-error.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+var assert = require('assert');
+var fs = require('fs');
+
+try {
+  fs.watch('non-existent-file');
```

Comment:
Can you use `assert.throws()`. If this doesn't throw, the test would happily pass.